substrate/test-utils/runtime/src/lib.rs

```diff
@@ -725,12 +728,44 @@ impl_runtime_apis! {
 	}
 
 	impl sp_genesis_builder::GenesisBuilder<Block> for Runtime {
-		fn create_default_config() -> Vec<u8> {
-			create_default_config::<RuntimeGenesisConfig>()
+		fn build_state(config: Vec<u8>) -> sp_genesis_builder::Result {
+			build_state::<RuntimeGenesisConfig>(config)
 		}
 
-		fn build_config(config: Vec<u8>) -> sp_genesis_builder::Result {
-			build_config::<RuntimeGenesisConfig>(config)
+		fn get_preset(params: &Option<PresetId>) -> Option<Vec<u8>> {
+			Some(if let Some(params) = params {
```

Comment:
I think you can un-nest this function once with `if let .. else`.

Reply:
reworked in a39c4bb